app/assets/v2/js/pages/profile-trust.js

```diff
@@ -774,4 +1056,76 @@ $(document).ready(function() {
     $(this).remove();
     $('#verify_offline_target').css('display', 'block');
   });
+
+  jQuery.fn.shake = function(interval, distance, times) {
+    interval = typeof interval == 'undefined' ? 100 : interval;
+    distance = typeof distance == 'undefined' ? 10 : distance;
+    times = typeof times == 'undefined' ? 3 : times;
+    var jTarget = $(this);
+
+    jTarget.css('position', 'relative');
+    for (var iter = 0; iter < (times + 1); iter++) {
+      jTarget.animate({ top: ((iter % 2 == 0 ? distance : distance * -1))}, interval);
+    }
+    return jTarget.animate({ top: 0}, interval);
+  };
+
+  $(document).on('click', '#gen_passport', function(e) {
+    e.preventDefault();
+    if (document.web3network != 'rinkeby') {
+      _alert('Please connect your web3 wallet to rinkeby + unlock it', 'error', 1000);
```

Comment:
should be rinkeby?

Reply:
yes, this is behind an `is_staff` flag + not in mainnet yet